docs/TBG_macros.cfg

```diff
@@ -91,6 +91,13 @@ TBG_gridDist="--gridDist '64{2}' '64,32{2},64'"
 TBG_periodic="--periodic 1 0 1"
 
 
+# Specifies boundary kinds for given species with a particle pusher, 'e' in the example.
+# The axis order matches --periodic.
+# Default: what is set by --periodic.
+# Currently only values that are same as default are supported, so this option can't change the behavior yet.
+TBG_boundaries="--e_boundary periodic absorbing periodic"
```

Comment:
Maybe `TBG_particleBoundaries` is better?

Reply:
This is only a doc variable, I will change in a follow-up PR with other work on particle boundaries